--- src/term.c	2018-05-10 21:39:42.000000000 +0900
+++ src/term.c	2018-05-10 21:39:42.000000000 +0900
@@ -2582,12 +2582,12 @@
 	out_flush();
 }
 
-static void out_char_nf(unsigned);
+static int out_char_nf(unsigned);
 
 /*
  * out_char_nf(c): like out_char(), but don't flush when p_wd is set
  */
-    static void
+    static int
 out_char_nf(unsigned c)
 {
 #if defined(UNIX) || defined(VMS) || defined(AMIGA) || defined(MACOS_X)
